### PR TITLE
Use Skeleton as ItemsList loading state

### PR DESCRIPTION
**Pull request by gabrieldutra**
## What type of PR is this? (check all applicable)

- [x] Other

## Description
I wanted to replace our Loading spinner for a while, so I'm exploring the option of using `Skeleton` for that (starting with ItemsList) and opening this for comments.

For now I'm keeping 10 rows and one thing I find a bit weird is when it's loading after we already had items in the screen (Changing sorting option for example). It does feel promising to be refined, so lmk what you think.

## Related Tickets & Documents
--

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
items-list-without-spinner

### CODE REVIEW

**Comment by gabrieldutra** [comment]
Alternatively without the spinner and only greying the results:

items-list-without-spinner

File: client/app/assets/less/inc/table.less [hunk 1]
```diff
@@ -1,149 +1,153 @@
...
-    tbody > tr > td {
-        padding-top: 5px !important;
-    }
-    
-    .btn-favourite, .btn-archive {
-        font-size: 15px;
-    }
+  thead > tr > th {
+    white-space: nowrap;
+  }
```
Comment (gabrieldutra) [inline client/app/assets/less/inc/table.less:100]: This is the only thing I added into this file, the rest is on Prettier

**Comment by gabrieldutra** [comment]
@arikfr @kravets-levko, this is certainly a start for having something other than the loading spinner we have. The grey loading when it already has loaded results is one of the things I most liked, it reduces sudden height changes and allows you to make page updates while loading.

The idea is to create a similar thing for the SettingsPages in a follow up, possibly allowing each column type to provide an specific skeleton loading (e.g.: if it's a button, show a button-skeleton) as [there are a few more options in antd v4](https://ant.design/components/skeleton/#components-skeleton-demo-element).

**Review by kravets-levko — APPROVED** [review]
Very nice @gabrieldutra! 🎉

**Pull request merged**